### PR TITLE
Tbt (tenant business type) from table to label

### PR DESCRIPTION
**Pull request by alextargov**
**Description**

Changes proposed in this pull request:
- Migrate tenant business type from table to application's label
- Remove graphql resolver and the whole tenant business type domain folder
- Add DB migrations

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: components/director/internal/systemfetcher/service.go [hunk 1]
```diff
@@ -354,19 +342,13 @@ func (s *SystemFetcher) processSystemsForTenant(ctx context.Context, tenantMappi
...
 			appInput, err := s.convertSystemToAppRegisterInput(ctx, system)
 			if err != nil {
 				return err
 			}
-			if tenantBusinessType != nil {
-				appInput.TenantBusinessTypeID = &tenantBusinessType.ID
-			}
+
+			log.C(ctx).Infof("Started processing tenant business type for system with system number %s", systemNumber)
+			appInput = s.enrichAppInputLabelsWithTenantBusinessType(systemPayload, appInput)
```
Comment (emmakarova) [inline components/director/internal/systemfetcher/service.go:351]: The system fetcher e2e tests can be extended with enriching the system payload for the mocked systems with the two properties and adding a check for them to be present in the created application labels.

**Review by emmakarova — APPROVED** [review]

**Review by PetarTodorovv — APPROVED** [review]
Formally approved, only go.mod/go.sum files are changed from our side

**Pull request merged**